package.json

```diff
@@ -10,11 +10,11 @@
   "types": "dist/index.d.ts",
   "scripts": {
     "build": "tsc",
+    "prepare": "npx install-peers",
```

Comment:
Double checking: does npx install-peers work for you?

Reply:
Worked for me:
```
~/cw/chain-events$ yarn
yarn install v1.21.1
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info fsevents@2.1.3: The platform "linux" is incompatible with this module.
info "fsevents@2.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
warning " > @istanbuljs/nyc-config-typescript@0.1.3" has unmet peer dependency "source-map-support@*".
[5/5] Building fresh packages...
$ npx install-peers
- Installing 4 peerDependencies...
+ Successfully installed 4 peerDependencies via yarn.

yarn add v1.21.1
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info fsevents@2.1.3: The platform "linux" is incompatible with this module.
info "fsevents@2.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
[5/5] Building fresh packages...
success Saved 16 new dependencies.
info Direct dependencies
info All dependencies
├─ @polkadot/keyring@6.5.1
├─ @polkadot/networks@6.5.1
├─ @polkadot/wasm-crypto-asmjs@4.0.2
├─ @polkadot/wasm-crypto-wasm@4.0.2
├─ @polkadot/wasm-crypto@4.0.2
├─ @polkadot/x-fetch@6.5.1
├─ @polkadot/x-randomvalues@6.5.1
├─ @polkadot/x-textdecoder@6.5.1
├─ @polkadot/x-textencoder@6.5.1
├─ @polkadot/x-ws@6.5.1
├─ @types/websocket@1.0.2
├─ cuint@0.2.2
├─ ip-regex@4.3.0
├─ rxjs@6.6.7
├─ scryptsy@2.1.0
└─ xxhashjs@0.2.2
$ npx install-peers
Only run install-peer-deps once. Skipping.
Done in 15.09s.
Done in 22.48s.
```